docs/static-pvc.md

```diff
@@ -117,3 +123,114 @@ persistentvolumeclaim/fs-static-pvc created
 
 **Note** deleting PV and PVC doesnot deleted the backend rbd image, user need to
 manually delete the rbd image if required
+
+## CephFS static PVC
+
+CephFS volumes created manually can be mounted and unmounted to an app, below step
+shows how to create a cephFS volume, static PV and static PVC
```

Comment:
@Madhu-1 I dont think we need to change it, `CephFS` is the correct usage.

Reply:
Its not cephfs volume it's cephfs subvolume isn't it

Reply:
@Madhu-1 @agarwal-mudit yeah, subvolume is correct, but I was mainly sharing the thought on the usage of "CephFS" and capitalization in it instead of `cephFS" as in this PR which looks wrong to me. @nixpanic @batrick what's the correct usage?  Is it `CephFS` or `cephFS`" ,that could help us to have a consistent mention of it in the repo.

Reply:
"CephFS" is correct